index.js

```diff
@@ -1,82 +1,66 @@
 'use strict';
 const {detailedDiff} = require('deep-object-diff');
-
-module.exports = options => {
-	options = {
-		isEnabled: true,
-		logStateChanges: true,
-		...options
```

Comment:
Just a note that this errors out in create react app: 

<img width="446" alt="screen shot 2018-04-12 at 4 57 38 pm" src="https://user-images.githubusercontent.com/236943/38709953-a38e5ee8-3e72-11e8-9fef-819de7685d5a.png">

Reply:
that is deleted code

Reply:
If this PR for whatever reason doesn't get merged I wanted to point this out for the library author